dhis-2/dhis-api/src/main/java/org/hisp/dhis/dataset/Section.java

```diff
@@ -134,6 +137,55 @@ public boolean hasDataElements()
         return dataElements;
     }
 
+    public Map<String, Collection<DataElement>> getOrderedDataElementsByCategoryCombo()
```

Comment:
I think perhaps move this to a new class like `SectionUtils` ? Might be better than adding a long business logic method to a domain class.

Reply:
this is now `org.hisp.dhis.dxf2.util.SectionUtils`